### ISSUE TITLE
few basic issues

### ISSUE THREAD

**Issue by ParthNema13** [mryashsinghal/Resume-Builder#53]
Extra Closing </div>

At the end of your .banner section:

</div> 
Fix: Remove this redundant </div> right after the .content div ends.

Missing alt Attributes

Some img tags lack meaningful alt text:

<img src="img/University-Student-Resume-Template-web1.jpg" alt="">
Fix:
<img src="img/University-Student-Resume-Template-web1.jpg" alt="University Student Resume Template">
Incorrect Button Tag

This is malformed:
<button onclick="document.location='resume.html'"></button class="select">Select</button>
Fix:

<button class="select" onclick="document.location='resume.html'">Select</button>

**Comment by mryashsinghal** [comment]
@Harishchandra-26, you can start working on this
